.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,37 @@
+name: build
+
+# build on c/cpp changes or workflow changes
```

Comment:
well in this case it'd make sense to run it on `builtin/**.lua` changes too
(and the `util/buildbot/**` one is irrelevant here)